server/routes/login.ts

```diff
@@ -128,7 +112,9 @@ export default function loginRouter(userAuthenticationService: UserAuthenticatio
         res.redirect(`/calendar/${utilities.getStartMonth()}#today`)
       }
     } catch (error) {
-      processError(error, req, res, userNotSignedUpMessage)
+      const sanitisedError = sanitiseError(error)
+      log.error(sanitisedError, `Unexpected login error, user = ${res.locals.user}`)
+      next(sanitisedError)
```

Comment:
The userNotSignedUp page is no longer applicable for errors, but will keep it for the edge case of user pressent in db but no email nor sms (this is true for about 45 users currently in db)